diff --git a/MicroBo/website/templates/index.html b/MicroBo/website/templates/index.html
new file mode 100644
index 0000000..873b7ef
--- /dev/null
+++ b/MicroBo/website/templates/index.html
@@ -0,0 +1,4 @@
+{% extends "base.html" %} {% block title %}Undone{% endblock %} {% block content %}
+<h1>Welcome to SSTI</h1>
+{{content}}
+{% endblock %}
\ No newline at end of file
diff --git a/MicroBo/website/views.py b/MicroBo/website/views.py
index 43bcf8e..bff5536 100644
--- a/MicroBo/website/views.py
+++ b/MicroBo/website/views.py
@@ -177,3 +177,10 @@ def delete_note():
             db.session.commit()
 
     return jsonify({})
+
+
+@views.route("/index/")
+def ssti():
+    # try to access 'http://127.0.0.1:5000/index/?content=<script>alert(/xss/)</script>'
+    content = request.args.get("content")
+    return render_template_string(content)
